clowdapp.yaml

```diff
@@ -245,6 +245,8 @@ objects:
           value: "${REDIS_SSL}"
         - name: SETTINGS__REDIS_CACHE_SSL
           value: "${REDIS_SSL}"
+        - name: MALLOC_ARENA_MAX
+          value: '2'
```

Comment:
This could've used a parameter for better customization if needed in the future rather than a hardcoded value. Perhaps in a follow-up PR ?

Reply:
I don't think that's needed at all, also it's a bit more flexible to experiment on stage by not messing around with app-interface.